client/containers/TopNavBarContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Hey! This file does not exist anymore / has been renamed. Do you know why you have it?